src/onnxflow/justbuildit/ignition.py

```diff
@@ -290,6 +277,9 @@ def load_or_make_state(
     model: build.UnionValidModelInstanceTypes = None,
     inputs: Optional[Dict[str, Any]] = None,
     quantization_samples: Optional[Collection] = None,
+    state_type: Type = build.State,
+    cache_validation_func: Callable = validate_cached_model,
```

Comment:
Allow frameworks built on top of `onnxflow` to supply their own cache validation function. For example, `groqflow` will call `onnxflow`'s validation function and then add a few extra rules of its own.